isis/src/control/apps/jigsaw/jigsaw.cpp

```diff
@@ -55,6 +62,72 @@ namespace Isis {
 
   void jigsaw(UserInterface &ui, Pvl *log) {
 
+    // <------- CSMState Output Testing
+    std::cout << "[jigsaw] Getting ISD via ALE" << std::endl;
+    QString fromlist = ui.GetFileName("FROMLIST");
+    SerialNumberList *snList = new SerialNumberList(fromlist);
+    QString filename0 = snList->fileName(0); // <-- testing with one file
+    
+    // Set ALESPICEROOT for now, when ALE SpiceQL Pr is merged, remove?
+    setenv("ALESPICEROOT", "/Volumes/t7-shield/isis_data", true);
+    json props;
+    props["web"] = true;
+    json isd = ale::load(filename0.toStdString(), props.dump(), "isis", true, false, false);
```

Comment:
This line should be:
```
json isd = ale::load(filename0.toStdString(), props.dump(), "ale", true, true, false);
```

The "ale" format is used in both ISIS and USGSCSM, then we want to set the first true so that we don't generate new spice data but read existing data from the ISIS cube